template/bridge/rtc/webNg/RtcEngine.ts

```diff
@@ -521,6 +523,29 @@ export default class RtcEngine {
     return devices;
   }
 
+  async setClientRole(
+    clientRole: ClientRole,
+    options?: ClientRoleOptions,
+  ): Promise<void> {
+    try {
+      if (clientRole == ClientRole.Audience) {
```

Comment:
What's the use of return, also use the defined Role enum
```suggestion
if (clientRole == ClientRole.Audience) {
        await this.client.setClientRole(Role.Audience, {level: 1});
      }
       if (clientRole == ClientRole.Broadcaster) {
        await this.client.setClientRole(Role.host);
        return;
      }
```